crates/uv-configuration/src/trusted_host.rs

```diff
@@ -25,11 +25,10 @@ impl TrustedHost {
             return false;
         }
 
-        if Some(self.host.as_ref()) != url.host_str() {
-            return false;
-        }
+        let allow_all_hosts = self.host == "*";
```

Comment:
Can we avoid the subsequent comparison in this case? i.e. just do `if self.host == "*" { return true }`?

Reply:
For sure — in fact I’d like to go one step further and make TrustedHost an enumerated with a “Wildcard” variant if y’all like that direction.

Reply:
> I’d like to go one step further and make TrustedHost an enumerated with a “Wildcard” variant

I ended up doing just that.